crates/bevy_text/src/pipeline.rs

```diff
@@ -53,6 +56,23 @@ pub struct TextPipeline {
     swash_cache: SwashCache,
 }
 
+impl TextPipeline {
+    /// Create text pipeline providing a flag for loading system fonts
+    pub fn new(load_system_fonts: bool) -> Self {
```

Comment:
`TextPipeline::new(true)` is very unreadable imo. Rename this to something like `with_load_system_fonts`.